spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2647Test.java

```diff
@@ -1,22 +1,15 @@
 package edu.umd.cs.findbugs.detect;
 
-import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
-import static org.hamcrest.MatcherAssert.assertThat;
 
 import org.junit.jupiter.api.Test;
 
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
 class Issue2647Test extends AbstractIntegrationTest {
     @Test
     void testIssue() {
         performAnalysis("ghIssues/Issue2647.class");
 
-        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder()
-                .build();
-
-        assertThat(getBugCollection(), containsExactly(0, bugTypeMatcher));
+        assertNoBugType("DMI_DOH");
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/2647) is about DMI_DOH FP.